code/graph_algorithms/src/Bipartite_check/bipartite.cpp

```diff
@@ -0,0 +1,49 @@
+#include<bits/stdc++.h>
+using namespace std;
```

Comment:
Don't use this; bad practice.

Reply:
Why did you unresolve this? It _is_ bad practice to use this line; see [here](https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice).